redshells/model/graph_convolutional_matrix_completion.py

```diff
@@ -302,49 +395,46 @@ def get_item_feature(self, user_ids: List, item_ids: List, additional_dataset: G
         dataset = self.graph_dataset.add_dataset(additional_dataset, add_item=True)
         return self._get_feature(user_ids=user_ids, item_ids=item_ids, with_user_embedding=with_user_embedding, graph=self.graph, dataset=dataset, session=self.session, feature='item')
 
-    @classmethod
-    def _predict(cls, user_ids: List, item_ids: List, with_user_embedding, graph: GraphConvolutionalMatrixCompletionGraph, dataset: GcmcGraphDataset,
+    def _predict(self, user_ids: List, item_ids: List, with_user_embedding, graph: GraphABC, dataset: GcmcGraphDataset,
                  session: tf.Session) -> np.ndarray:
         if graph is None:
             RuntimeError('Please call fit first.')
 
         rating_adjacency_matrix = dataset.train_rating_adjacency_matrix()
         user_indices, item_indices = dataset.to_indices(user_ids, item_ids)
         if not with_user_embedding:
-            user_indices = np.array([0] * len(user_indices))  # TODO use default user index.
+            user_indices = np.array([0] * len(user_indices))
 
         user_feature_indices, item_feature_indices = dataset.to_feature_indices(user_ids, item_ids)
         input_data = dict(user=user_indices, item=item_indices, user_feature_indices=user_feature_indices, item_feature_indices=item_feature_indices)
-        feed_dict = cls._feed_dict(input_data, graph, dataset, rating_adjacency_matrix, is_train=False)
+        feed_dict = self._feed_dict(input_data, graph, dataset, rating_adjacency_matrix, is_train=False)
```

Comment:
@mski-iksm 
It might be better to leave `_predict()` function to be class method, because class method prevents the `_predict()` function from having access to the class attributes and this might lead your code more readable.

Reply:
I've changed `_predict()` to classmethod and `_feed_dict()` to staticmethod